### PR TITLE
Add integration tests to nightly test suite

### PR DESCRIPTION
**Pull request by richard-rogers**
## Description

Includes integration tests in nightly test run.

- [ ] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by jamie256 — DISMISSED** [review]
LGTM, lets try it out

File: docker_test.sh [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 git config --global --add safe.directory /workspace
 export PATH=$PATH:/home/whyuser/.local/bin
+WHYLABS_DEFAULT_DATASET_ID=model-2250
+WHYLABS_DEFAULT_ORG_ID=org-0
```
Comment (jamie256) [inline docker_test.sh]: I don't think we need an org_id anymore, right?

Comment (richard-rogers) [inline docker_test.sh]:
```suggestion
'```

File: docker_test.sh [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 git config --global --add safe.directory /workspace
 export PATH=$PATH:/home/whyuser/.local/bin
+WHYLABS_DEFAULT_DATASET_ID=model-2250
+'```
```
Comment (richard-rogers) [inline docker_test.sh]:
```suggestion
```

**Review by jamie256 — APPROVED** [review]

**Pull request merged**